packages/codemod/bin/cli.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const mri = require('mri');
+const glob = require('glob');
+const { run: jscodeshift } = require('jscodeshift/src/Runner');
+
+const flags = mri(process.argv.slice(2), {
+  alias: { help: ['h'] },
+  boolean: ['dry-run'],
+});
+const commands = flags._;
+
+if (commands.length === 0 || (flags.help && commands.length === 0)) {
+  console.log(`
+Usage: mc-codemod [global-options] [transform] [glob-pattern]
+
+Global options:
+
+  --dry-run             (optional) Executes the command but does not send any mutation request.
+
+Transforms:
+
+  rename-js-to-jsx       Rename ".js" files using React JSX syntax to ".jsx".
+  `);
+  process.exit(0);
+}
+
+const [transform, globPattern] = commands;
+
+const files = glob.sync(globPattern);
+
+switch (transform) {
+  case 'rename-js-to-jsx':
+    break;
```

Comment:
Caught me for a minute here but I guess it's just a `throwIfUnknownTransform` sort of.

Reply:
I was also scratching my head for a couple of minutes 😄

Reply:
Then maybe dumping this into such named function isn't the worst idea 😄.

Reply:
Yes it's just to validate that the command is valid, since we're using the name to read from file system.

Should I make this more explicit?

Reply:
My suggestion would be something like this:

```
const KNOWN_TRANSFORMATIONS = [
  'rename-js-to-jsx'
];

if (!KNOWN_TRANSFORMATIONS.includes(transform) {
  throw new Error(`Unknown transform ${transform}.`);
}

Reply:
Yes, like

```js
function throwIfTransformIsUnknown() {
const KNOWN_TRANSFORMATIONS = [
  'rename-js-to-jsx'
];

if (!KNOWN_TRANSFORMATIONS.includes(transform) {
  throw new Error(`Unknown transform ${transform}.`);
}
}
```